system-int-test/src/cmr/system_int_test/data2/umm_spec_service.clj

```diff
@@ -71,3 +72,111 @@
          (service attribs)
          (assoc :provider-id provider-id :native-id native-id)
          umm-s->concept))))
+
+(defn contact-person
+  "Returns a ContactPersonType suitavke as an element in a
```

Comment:
You got lucky that was a real word.

Reply:
I should really update that docstring to make it more clear. Maybe just a parenthetical is needed: `Returns a ContactPersonType suitavke (Bosnian for "suitecases") as an element`

Reply:
Okay, "suitcases" has been removed, replaced with something more suitable.